nilmtk/siteonlyapi.py

```diff
@@ -0,0 +1,383 @@
+from nilmtk.dataset import DataSet
+from nilmtk.metergroup import MeterGroup
+import pandas as pd
+from nilmtk.losses import *
+import numpy as np
+import matplotlib.pyplot as plt
+import datetime
+from IPython.display import clear_output
+
+
+class API():
```

Comment:
You commit a new file that declares the `API` class a second time.
Instead of duplicating code, it would be better that you extend the existing `api.API` class with your use case.

Reply:
makes sense.. alright .I'll do that

Reply:
I have extended the `API` class as you recommended which requires in the experiment parameters to be mentioned if its site meter only.